engine/template.go

```diff
@@ -119,7 +122,22 @@ func (t *Templates) renderHTMLTemplate(key TemplateKey, breadcrumbs []Breadcrumb
 
 func (t *Templates) renderNonHTMLTemplate(key TemplateKey, params interface{}) {
 	file := filepath.Clean(filepath.Join(key.Directory, key.Name))
-	compiled := texttemplate.Must(texttemplate.New(filepath.Base(file)).Funcs(combinedFuncs).ParseFiles(file))
+	var compiled *texttemplate.Template
+	if _, err := os.Stat(file); os.IsNotExist(err) {
```

Comment:
het voelt wat complex aan, met 2x een os.Stat. Kunnen we deze flow wat vereenvoudigen? Anders even samen naar kijken.